src/pages/loading-waiting-states/progress-bar/index.tsx

```diff
@@ -152,17 +136,14 @@ export const ProgressBar = (): JSX.Element => {
         setFileUploadList((oldList) => [...oldList, createFileItem(nextFileIndex++)]);
     }, [fileUploadList, setFileUploadList]);
 
-    const removeListItem = useCallback((id: number, status: string): void => {
-        if (status === 'Complete') {
+    const markUploadComplete = useCallback((id: number, status: string, reason?: string): void => {
+        if ((reason && reason === 'clickaway') || status === 'complete') {
```

Comment:
What is this check intending to do? `status` can never === 'complete' (lowercase)

Reply:
My bad. This needs to updated as `Complete`. The reason I am checking status is If user clicks on View button then it should not delete InfoListItem. (Since file has already uploaded and waiting for timeout )